test/infrastructure/docker/api/v1beta1/devmachine_types.go

```diff
@@ -85,6 +85,61 @@ const (
+
+// ContainerProvisioned condition and corresponding reasons that will be used in v1Beta2 API version for a DevMachine's docker backend.
+const (
+	// DevMachineDockerContainerProvisionedV1Beta2Condition documents the status of the provisioning of the container
+	// generated by a DevMachine's docker backend.
+	//
+	// NOTE as a difference from other providers, container provisioning and bootstrap are directly managed
+	// by the DevMachine's docker backend controller (not by cloud-init).
+	DevMachineDockerContainerProvisionedV1Beta2Condition string = "ContainerProvisioned"
+
+	// DevMachineDockerContainerWaitingForClusterInfrastructureV1Beta2Reason documents the container for a DevMachine's docker backend waiting for the cluster
+	// infrastructure to be ready.
+	DevMachineDockerContainerWaitingForClusterInfrastructureV1Beta2Reason = clusterv1.WaitingForClusterInfrastructureReadyV1Beta2Reason
+
+	// DevMachineDockerContainerWaitingForControlPlaneInitializedV1Beta2Reason documents a container for a DevMachine's docker backend waiting
+	// for the control plane to be initialized.
+	DevMachineDockerContainerWaitingForControlPlaneInitializedV1Beta2Reason = clusterv1.WaitingForControlPlaneInitializedV1Beta2Reason
+
+	// DevMachineDockerContainerWaitingForBootstrapDataV1Beta2Reason documents a container for a DevMachine's docker backend waiting for the bootstrap
+	// data to be ready.
+	DevMachineDockerContainerWaitingForBootstrapDataV1Beta2Reason = clusterv1.WaitingForBootstrapDataV1Beta2Reason
+
+	// DevMachineDockerContainerProvisionedV1Beta2Reason documents the container for a DevMachine's docker backend is provisioned.
+	DevMachineDockerContainerProvisionedV1Beta2Reason = clusterv1.ProvisionedV1Beta2Reason
+
+	// DevMachineDockerContainerWaitingForClusterInfrastructureV1Beta2Reason documents the container for a DevMachine's docker
+	// backend is not provisioned.
+	DevMachineDockerContainerNotProvisionedV1Beta2Reason = clusterv1.NotProvisionedV1Beta2Reason
+
+	// DevMachineDockerContainerDeletingV1Beta2Reason surfaces when the container for a DevMachine's docker
+	// backend is deleting.
+	DevMachineDockerContainerDeletingV1Beta2Reason = clusterv1.DeletingV1Beta2Reason
+)
+
+// BootstrapExecSucceeded condition and corresponding reasons that will be used in v1Beta2 API version for a DevMachine's docker backend.
+const (
+	// DockerMachineBootstrapExecSucceededV1Beta2Condition provides an observation of the DevMachine's docker backend bootstrap process.
+	// It is set based on successful execution of bootstrap commands and on the existence of
+	// the /run/cluster-api/bootstrap-success.complete file.
+	// The condition gets generated after ContainerProvisionedCondition is True.
+	//
+	// NOTE as a difference from other providers, container provisioning and bootstrap are directly managed
+	// by the DevMachine's docker backend controller (not by cloud-init).
+	DevMachineDockerBootstrapExecSucceededV1Beta2Condition string = "BootstrapExecSucceeded"
```

Comment:
```suggestion
	DevMachineDockerContainerBootstrapExecSucceededV1Beta2Condition string = "BootstrapExecSucceeded"
```
Maybe, all the ones above have that prefix. If yes, same for the two below